layouts/webinar/single.html

```
@@ -3,162 +3,194 @@
+
+    <!-- No Webinar Planned: Newsletter Signup -->
+    <section x-show="webinar.status && webinar.status !== 'Planned'" x-data="{data: {email: '', listId: 'cryptomatorhub', captcha: null, acceptTerms: false, errorMessage: '', success: false, inProgress: false}, captchaState: null}" class="container py-12 text-center" x-cloak>
+      <header class="mb-8">
+        <h1 class="font-h1 mb-4">{{ i18n "webinar_none_title" }}</h1>
+        <p class="lead max-w-2xl mx-auto">{{ i18n "webinar_none_lead" }}</p>
+      </header>
+
+      <form x-show="!data.success" x-ref="form" class="max-w-xl mx-auto mb-4" @submit.prevent="new Newsletter($refs.form, data).subscribe(); $refs.captcha.reset()">
+        <p class="font-p mb-4">{{ i18n "webinar_none_newsletter_instruction" }}</p>
+        <div class="flex justify-center items-center rounded-sm bg-gray-300 mb-4">
+          <input x-model="data.email" x-ref="email" @blur="$refs.email.classList.add('show-invalid')" type="email" class="grow input-box rounded-r-none focus:border-secondary" placeholder="{{ i18n "hub_newsletter_email_placeholder" }}" required/>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)